src/views/Patient/Patient.tsx

```diff
@@ -0,0 +1,57 @@
+import './Patient.css';
+import { Box, Card, CardContent, Typography } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+function Patient(props: any) {
```

Comment:
define an interface for the props.  I'd also recommend maybe renaming this file so as not to conflict with the `Patient` type.